components/wpt-results.html

```diff
@@ -138,12 +138,20 @@
           testRuns: {
             type: Array,
           },
+          filteredDirectories: {
+            type: Array,
+            value: []
```

Comment:
Ah, got it. Optionally could add a short comment here saying that the displayed specs should be the union of the filtered directories (not the intersection).